kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabTcpIpUi.ui.xml

```diff
@@ -89,7 +89,7 @@
                             </b:FormGroup>
 
                             <b:FormGroup>
-                                <b:Button ui:field="renew" />
+                                <b:Button ui:field="renew" addStyleNames="fa fa-refresh"/>
```

Comment:
Pleae change header to: "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"